container-compose.yml

```diff
@@ -185,6 +185,7 @@ services:
         - orbit_source=./orbit
     environment:
       TZ: ${SINGULARITY_TIMEZONE}
+      HOSTNAME: ${SINGULARITY_HOSTNAME}
```

Comment:
I thought we had talked about moving this to a config.py file. At the very least, I would use an env var within the container like `SINGULARITY_HOSTNAME` that does not collide with and existing one set by podman

Reply:
Moving to a config.py could be part of a later change to move all hostnames to a config.py, but for now I just did it the same way we do it for orbit earlier in the file, which must also conflict with the $HOSTNAME set by podman but has done so for some time now.